src/mappings/staking.ts

```diff
@@ -70,6 +95,11 @@ export async function handleSetCommissionRate(
   const { valueAt, valueAfter } = commissionRates;
 
   if (valueAt !== valueAfter) {
-    await updateIndexerChallenges(indexer, 'CHANGE_COMMISSION', event.blockTimestamp);
+    await updateIndexerChallenges(
+      indexer,
+      'CHANGE_COMMISSION',
+      event.blockTimestamp,
```

Comment:
[suggestion] As you want both `blockTimestamp` and `blockNumber` for updateChallenges.
Could you try to use one param `event` other than passing them separately

Reply:
Fixed this